uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/LicenseTest.java

```diff
@@ -71,7 +71,8 @@
 			}
 		}
 		new PushButton("Close").click();
-		;
+		new DefaultShell("About Red Hat JBoss Developer Studio");
+		new PushButton("Close").click();
```

Comment:
There are two windows with the same button "Close". To be 100% sure we click on the correct button we can use reference composite, e.g.
```
shell = new DefaultShell("About Red Hat JBoss Developer Studio");
new PushButton(shell, "Close").click();
```
Please use the above approach whenever you can.